cs/v1/CS_v1.0.0.yaml

```diff
@@ -514,12 +512,133 @@ paths:
+            - `BORD` (Border)
+            - `CLOC` (Customer Location)
+            - `COFS` (Container Freight Station)
+            - `OFFD` (Off Dock Storage)
+            - `DEPO` (Depot)
+            - `INTE` (Inland Terminal)
+            - `POTE` (Port Terminal)
+            - `PBPL` (Pilot Boarding Place)
+            - `BRTH` (Berth)
+            - `RAMP`  (Ramp)
+            - `WAYP` (Waypoint)
+          example: POTE
+          type: string
+        location:
+          oneOf:
+            - $ref: '#/components/schemas/UNLocationLocation'
+            - $ref: '#/components/schemas/FacilityLocation'
+            - $ref: '#/components/schemas/AddressLocation'
+          discriminator:
+            propertyName: locationType
+            mapping:
+              ADDR: '#/components/schemas/AddressLocation'
+              FACS: '#/components/schemas/FacilitySMDGLocation'
+              UNLO: '#/components/schemas/UNLocationLocation'
+        dateTime:
+          type: string
+          format: date-time
+          example: '2025-01-14T09:21:00+01:00'
+          description: 'The local date and time, when the event will take place, in ISO 8601 format specifying the offset.'
+    Arrival:
+      title: Place of Arrival
+      type: object
+      description: The Location specifying where the place of arrival is located.
+      required:
+        - facilityTypeCode
+        - location
+        - dateTime
+      properties:
+        facilityTypeCode:
+          description: |
+            The code to identify the specific type of facility. The code indicates which role the facility plays during the transportCall.
+            - `BORD` (Border)
+            - `CLOC` (Customer Location)
+            - `COFS` (Container Freight Station)
+            - `OFFD` (Off Dock Storage)
+            - `DEPO` (Depot)
+            - `INTE` (Inland Terminal)
+            - `POTE` (Port Terminal)
+            - `PBPL` (Pilot Boarding Place)
+            - `BRTH` (Berth)
+            - `RAMP`  (Ramp)
+            - `WAYP` (Waypoint)
+          example: POTE
+          type: string
+        location:
+          oneOf:
+            - $ref: '#/components/schemas/UNLocationLocation'
+            - $ref: '#/components/schemas/FacilityLocation'
+            - $ref: '#/components/schemas/AddressLocation'
+          discriminator:
+            propertyName: locationType
+            mapping:
+              ADDR: '#/components/schemas/AddressLocation'
+              FACS: '#/components/schemas/FacilitySMDGLocation'
+              UNLO: '#/components/schemas/UNLocationLocation'
+        dateTime:
+          type: string
+          format: date-time
+          example: '2025-01-14T09:21:00+01:00'
+          description: 'The local date and time, when the event will take place, in ISO 8601 format specifying the offset.'
+    Departure:
+      title: Place Of Departure
+      type: object
+      description: The Location specifying where the place of departure is located.
```

Comment:
Please also add the `maxLength: 4` to `facilityTypeCode` further down (I cannot add the comment the exact place)